Add tailwind

novellac opened the pull request:
**Work done**

This PR adds the Tailwind packages, as per the [Girdsome Tailwind docs](https://next.gridsome.org/plugins/gridsome-plugin-tailwindcss), and the [Tailwind compat docs](https://tailwindcss.com/docs/installation#post-css-7-compatibility-build).

**Testing instructions**
Run gridsome locally (see package.json for scripts you can use).
Add a tailwind class to any element, and check it in the browser. The proper Tailwind styles should be applied.

novellac committed:
Add tailwind

alliequintano reviewed (approved):
Tested out a few different tailwind classes and it worked!

novellac committed:
design: Update icon, default layout

The default layout wraps both the index and the CardPost.

This styling will support card styling.

novellac committed:
bonus: Update index to remove gridsome boilerplate

novellac committed:
feat: add date to card page

novellac committed:
Merge pull request #9 from cherryontech/feat/date

feat: Add styling and date
